### ISSUE TITLE
About weight Mask

### ISSUE THREAD

**Issue by jnulzl** [YadiraF/PRNet#99]
@YadiraF Hi,for weight ratio=16:4:3:0  and the weight mask show be a matrix whos value only 16,4,3,0. right or wrong?

**Comment by huybery** [comment]
Hi, junlzl, thanks for your share training code, but i can't find weight mask loss function? I only find simply mean_squared_error.

**Comment by WWWJL** [comment]
> Hi, junlzl, thanks for your share training code, but i can't find weight mask loss function? I only find simply mean_squared_error.

嗨，这个问题您知道怎么回事了吗？求解答，感激
